FSQCellManifest/FSQCellManifest.h

```diff
@@ -449,32 +450,32 @@ extern const NSInteger kRowIndexForFooterIndexPaths;
  
  @return A new table view cell manifest.
  */
-- (instancetype)initWithDelegate:(id)delegate 
-                         plugins:(NSArray *)plugins
+- (instancetype)initWithDelegate:(nullable id)delegate
```

Comment:
This one is weird... we have several slightly different protocols that can be used for the delegate: FSQCellManifestRecordModificationDelegate, FSQCellManifestRecordConfigurationDelegate, FSQCellManifestRecordSelectionDelegate, and FSQCellManifestRecordSizingDelegate.

Maybe we should create a base FSQCellManifestDelegate that is empty and all of these delegates inherit from it?